bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/defmodel/ParameterItem.java

```diff
@@ -44,15 +44,16 @@ public class ParameterItem {
     private Validation validation;
     @Nullable
     private BigDecimal offset;
-    @Nullable
-    private Boolean isstr;
+    private Boolean isstr = false;
```

Comment:
I think it's more expressive not to initialize the properties with default values and make them @Nullable as they can be missing from the definition, which will be represented by a null.

Reply:
In my opinion, the issst variable is useless. Because the rule = 5 indicates, that this is a string.
and if it is not given, I assume the original author treats it as false as default. That's why I implemented it like this. But I can change it.